config/initializers/assets.rb

```diff
@@ -8,6 +8,6 @@
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-Rails.application.config.assets.precompile += ['about.css.sass', 'registration.css', 'report.css', '*.js', '*.coffee']
+Rails.application.config.assets.precompile += ['registration.css', 'report.css', '*.js', '*.coffee']
```

Comment:
Yep, I've done this in another project before. Thoughtbot provides their own default rubocop.yml https://github.com/thoughtbot/hound/blob/master/config/style_guides/ruby.yml